src/signals/incident-management/containers/StandardTextsAdmin/components/Subcategories/Subcategories.tsx

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import { useCallback, useMemo, useState } from 'react'
+
+import { Column, Row } from '@amsterdam/asc-ui'
+import { useSelector } from 'react-redux'
+import { useNavigate } from 'react-router-dom'
+
+import BackLink from 'components/BackLink'
+import CheckboxList from 'components/CheckboxList'
+import FormFooter from 'components/FormFooter'
+import Label from 'components/Label'
+import { makeSelectStructuredCategories } from 'models/categories/selectors'
+import type SubCategory from 'types/api/sub-category'
+
+import { CategoryColumns, StyledUnderline } from './styled'
+import PageHeader from '../../../../../../components/PageHeader'
+
+type Props = {
+  onChange: (categoryIds: number[]) => void
+  value: number[] | undefined
+  defaultText?: string
+}
+
+export const Subcategories = ({ onChange, value, defaultText }: Props) => {
+  const subCategories = useSelector(makeSelectStructuredCategories)
+  const [newValue, setNewValue] = useState<number[]>(value ?? [])
```

Comment:
maybe instead of checking if value exists here. Set value = [] in de props as default.

Reply:
don